inspirehep/dojson/jobs/fields/bd1xx.py

```diff
@@ -130,9 +130,3 @@ def ranks(self, key, value):
         for _rank in _ranks:
             self['_ranks'].append(_rank)
             self['ranks'].append(classify_rank(_rank))
-
-
-@jobs.over('experiment', '^693..')
-@utils.for_each_value
-def experiment(self, key, value):
-    return value.get('e')
```

Comment:
Because `experiment` is not a key in the schema: https://github.com/inspirehep/inspire-next/blob/303444d0e344d3a4c750d82c45c6094a6f224e05/inspirehep/modules/records/jsonschemas/records/jobs.json, and `693__e` is already consumed by rule `experiments`, which populates the right key `experiments`.